training_pipeline/pipeline/configs.py

```diff
@@ -25,6 +25,7 @@
 PIPELINE_ROOT = os.path.join(OUTPUT_DIR, "tfx_pipeline_output", PIPELINE_NAME)
 
 DATA_PATH = "gs://pets-tfrecords/pets-tfrecords/"
+SCHEMA_PATH = "pipeline/schema.pbtxt"
```

Comment:
Is it also possible to supply a GCS path here?

Reply:
not sure! maybe possible. need to check. 
will let you know soon the result

Reply:
Did you get a chance to see this?

Reply:
just checked right now, and it works with GCS as well :)